e2etest/go.mod

```diff
@@ -14,10 +19,10 @@ require (
 	github.com/prometheus/client_golang v1.19.0 // indirect
 	go.uber.org/zap v1.26.0 // indirect
 	golang.org/x/sys v0.18.0 // indirect
-	k8s.io/api v0.29.3
+	k8s.io/api v0.29.4
```

Comment:
is there a reason for these bumps?

Reply:
These are submitted with this: https://github.com/metallb/metallb/pull/2351#discussion_r1568464636

Reply:
@fedepaol So Should I downgrade this dependency version?